### DISCUSSION TITLE
1.5.2

### DISCUSSION THREAD

**Discussion by jwills** [duckdb/dbt-duckdb#192] [Announcements]
## What's Changed
* Prepping the dbt-duckdb tests to pass under the next release of DuckDB by @jwills in https://github.com/jwills/dbt-duckdb/pull/166
* For the remote.password stuff to work we need to use it with psycopg2.connect by @jwills in https://github.com/jwills/dbt-duckdb/pull/167
* Bump mypy from 1.2.0 to 1.3.0 by @dependabot in https://github.com/jwills/dbt-duckdb/pull/168
* Add a fast-path option for loading dbt's seed files via a DuckDB COPY operation by @jwills in https://github.com/jwills/dbt-duckdb/pull/169
* Revamp the plugins setup to support configuring the DuckDBPyConnection instance directly by @jwills in https://github.com/jwills/dbt-duckdb/pull/171
* Some fixes for catalog lookups in a multi-database, multi-schema world by @jwills in https://github.com/jwills/dbt-duckdb/pull/172
* Bump dbt-tests-adapter from 1.5.0 to 1.5.1 by @dependabot in https://github.com/jwills/dbt-duckdb/pull/175
* Create a simple plugin system for writing data to external destinations by @jwills in https://github.com/jwills/dbt-duckdb/pull/173
* Prevent loading source data via plugin during dbt parsing phase by @AlexanderVR in https://github.com/jwills/dbt-duckdb/pull/178
* Simplify extension loading in dbt-duckdb by @jwills in https://github.com/jwills/dbt-duckdb/pull/180
* Updates to support a broader class of incremental functionality by @jwills in https://github.com/jwills/dbt-duckdb/pull/181
* Prep for Python 3.7 EOL by @jwills in https://github.com/jwills/dbt-duckdb/pull/182
* Updates for working with MotherDuck databases by @jwills in https://github.com/jwills/dbt-duckdb/pull/183
* Automatically disable transactions on motherduck paths by @jwills in https://github.com/jwills/dbt-duckdb/pull/186
* Add an example of doing a write-side operation in the sqlalchemy plugin by @jwills in https://github.com/jwills/dbt-duckdb/pull/187
* Support alternate string formatting strategies for external sources by @jwills in https://github.com/jwills/dbt-duckdb/pull/188
* Disable this test for now while DuckDB issue 7934 gets worked out by @jwills in https://github.com/jwills/dbt-duckdb/pull/189
* Add in MotherDuck docs and do a version bump by @jwills in https://github.com/jwills/dbt-duckdb/pull/190
* Document the Plugin.store functionality in the README by @jwills in https://github.com/jwills/dbt-duckdb/pull/191

**Full Changelog**: https://github.com/jwills/dbt-duckdb/compare/1.5.1...1.5.2

<hr /><em>This discussion was created from the release <a href='https://github.com/jwills/dbt-duckdb/releases/tag/1.5.2'>1.5.2</a>.</em>

**Comment by kokorin** [comment]
dbt-duckdb 1.5.2 has a tiny (but a bit annoying) bug fixed in #204 but not backported to 1.5.x. 
Considering that dbt-core 1.5 reached end of life, do you plan to support it further and release 1.5.3?
